### PR TITLE
chore: simplified if statement

### PR DESCRIPTION
**Pull request by hayk96**
## Description
Changes that have been made are minor. Code style changes.

## Changes Made
- Simplified `if` statement. Multiple "not and" logical operators have been replaced with  `any()` function

## Checklist
- [x] I have read and followed the contributing guidelines.
- [x] I have tested my changes thoroughly and they work as expected.
- [ ] I have added necessary tests for the changes made.
- [ ] I have updated the documentation to reflect the changes made.
- [ ] My code follows the project's coding style and standards.
- [x] I have added appropriate commit messages and comments for my changes.

### CODE REVIEW

File: src/gcp_scanner/arguments.py [hunk 1]
```diff
@@ -127,9 +127,12 @@ def arg_parser():
 
   args: argparse.Namespace = parser.parse_args()
 
-  if not args.key_path and not args.gcloud_profile_path \
-    and not args.use_metadata and not args.access_token_files\
-    and not args.refresh_token_files:
+  if not list(filter(lambda arg: bool(arg) is True,
```
Comment (ZetaTwo) [inline src/gcp_scanner/arguments.py:130]: Please use a list comprehension instead of filter+lambda

File: .gitignore [hunk 2]
```diff
@@ -166,4 +166,7 @@ cython_debug/
 #  option (not recommended) you can uncomment the following to ignore the entire idea folder.
 #.idea/
 
+# OS specific
+.DS_Store
```
Comment (ZetaTwo) [inline .gitignore:170]: Put this in your own global gitignore instead of here.

**Review by ZetaTwo — CHANGES_REQUESTED** [review]

File: src/gcp_scanner/arguments.py (hunk 1 above)
Comment (hayk96) [reply src/gcp_scanner/arguments.py]: good point

File: src/gcp_scanner/arguments.py [hunk 3]
```diff
@@ -127,9 +127,13 @@ def arg_parser():
 
   args: argparse.Namespace = parser.parse_args()
 
-  if not args.key_path and not args.gcloud_profile_path \
-    and not args.use_metadata and not args.access_token_files\
-    and not args.refresh_token_files:
+  if not [arg for arg in
```
Comment (ZetaTwo) [inline src/gcp_scanner/arguments.py:130]: Pretty good but can be even clearer with:
`if not any(bool(arg) for arg in [...]):`

**Review by ZetaTwo — CHANGES_REQUESTED** [review]

File: .gitignore [hunk 4]
```diff
@@ -166,4 +166,4 @@ cython_debug/
 #  option (not recommended) you can uncomment the following to ignore the entire idea folder.
 #.idea/
 
-# End of https://www.toptal.com/developers/gitignore/api/python
+# End of https://www.toptal.com/developers/gitignore/api/python
```
Comment (ZetaTwo) [inline .gitignore:169]: Please keep newline at the end

File: src/gcp_scanner/arguments.py [hunk 5]
```diff
@@ -127,9 +127,12 @@ def arg_parser():
 
   args: argparse.Namespace = parser.parse_args()
 
-  if not args.key_path and not args.gcloud_profile_path \
-    and not args.use_metadata and not args.access_token_files\
-    and not args.refresh_token_files:
+  if not any([arg for arg in
```
Comment (ZetaTwo) [inline src/gcp_scanner/arguments.py:130]: No need to do `[x for x in Y]`, you  can replace it with simply `Y`

**Review by ZetaTwo — CHANGES_REQUESTED** [review]
Small things

**Review by ZetaTwo — APPROVED** [review]

**Pull request merged**